### PR TITLE
Move Connecting Cloud Storage out of Advanced Usage

### PR DESCRIPTION
**Pull request by munkyshi**
### Linked issue(s):
None

### What change does this PR introduce and why?

Move Connecting Cloud Storage out of Advanced Usage and move Advanced Usage into workflow

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: docs/workflow/advanced-usage/index.md [hunk 1]
```diff
@@ -9,35 +9,28 @@ hide:
 This section contains tutorial documentation for advanced features available in Kolena.
```
Comment (ly-kolena) [inline docs/workflow/advanced-usage/index.md]: Can be updated to `advanced features for Kolena workflow`

**Comment by ly-kolena** [comment]
also https://github.com/kolenaIO/kolena/pull/415 ? If this PR gets green checks first, I'll close mine.

**Review by ly-kolena — APPROVED** [review]

**Comment by ly-kolena** [comment]
> also #415 ? If this PR gets green checks first, I'll close mine.

Realized the structural differences between 2 PRs, will close mine.

**Review by gordonhart — APPROVED** [review]
Looks good. This does promote the cloud storage concept to be equal with the "Datasets" and "Workflows" sections, but that might be a good thing, given how frequently this part of the docs is accessed.

**Pull request merged**